concrete/single_pages/dashboard/system/basics/icons.php

```diff
@@ -16,7 +16,7 @@
 
     <fieldset>
         <legend><?=t('iPhone Thumbnail')?></legend>
-        <div class="help-block"><?=t('iPhone home screen icons should be 57x57 and be in the .png format.')?></div>
+        <div class="help-block"><?=t('iPhone home screen icons should be 180x180 and be in the .png format.')?></div>
```

Comment:
What about
```php
t('iPhone home screen icons should be %1$dx%2$d and be in the %3$s format.', 180, 180, '.png')
```
that way, we (translators) won't have to translate it again if things changes again...
(the same for the other changed strings - not for the unchanged ones)